docs/experimenter/openapi-schema.json

```diff
@@ -1613,6 +1613,39 @@
         ]
       }
     },
+    "/api/v6/nimbus-fml-diagnostics": {
+      "post": {
+        "operationId": "createNimbusFmlDiagnosticsApi",
+        "description": "",
+        "parameters": [],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {}
+            },
+            "application/x-www-form-urlencoded": {
+              "schema": {}
+            },
+            "multipart/form-data": {
+              "schema": {}
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "content": {
+              "application/json": {
+                "schema": {}
+              }
+            },
+            "description": ""
+          }
+        },
+        "tags": [
+          "Nimbus: Public"
+        ]
+      }
+    },
```

Comment:
I don't know if these docs files are just docs, or whether they're significant to the working of the code.

If they are the latter, then filing in the `schema` objects may be the missing piece to what we discussed yesterday. 

Either way, they endpoint should be documented before landing this PR.